app/src/screens/ProposalDetail.js

```diff
@@ -198,10 +282,46 @@ function ProposalDetail({ proposal, onBack, requestToken }) {
                         lines={chartLines}
                       />
                     </div>
+
+                    {mode === 'update' && (
+                      <Field label="Amount of your tokens for this proposal">
+                        <div
+                          css={`
+                            display: flex;
+                            justify-content: space-between;
+                          `}
+                        >
+                          <div
+                            css={`
+                              width: 100%;
+                            `}
+                          >
+                            <Slider
+                              css={`
+                                padding-left: 0px;
+                                padding-right: ${2 * GU}px;
+                              `}
+                              value={progress}
+                              onUpdate={setProgress}
+                            />
+                          </div>
+                          <TextInput
+                            value={inputValue}
+                            onChange={setAmount}
+                            type="number"
+                            max={maxAvailable}
+                            min={'0'}
+                            required
+                            ref={inputRef}
```

Comment:
Perhaps we should seet a fixed width for this input

For me is showing 

![Screen Shot 2020-06-23 at 5 38 51 PM](https://user-images.githubusercontent.com/22663232/85459536-a3fa1500-b578-11ea-8115-62179f85b831.png)